### DISCUSSION TITLE
Sugestão de nova redação para dicas dos botões

### DISCUSSION THREAD

**Discussion by ronaldosc** [UNIVALI-LITE/Portugol-Studio#1006] [General]
https://github.com/UNIVALI-LITE/Portugol-Studio/blob/4ddec5e32e27e279a7b02fc71d1057295e18c405/ide/src/main/resources/br/univali/ps/ui/imagens/dicas/index.properties#L36

Sugiro uma nova redação para explicar a função dos botões:
- "Botão Depurar": Executa o código linha a linha por vez com pausas.
- "Botão Cancelar": Interrompe a execução do código.

**Comment by AdsonEsteves** [comment]
@noschang 
O que acha? Parece uma boa.
